.storybook/preview-head.html

```diff
@@ -0,0 +1,13 @@
+<!-- Pulling in global styles from external sources -->
+<link
+  rel="stylesheet"
+  data-entry-name="style.css"
+  href="https://unpkg.com/@department-of-veterans-affairs/formation/dist/formation.min.css"
+/>
+
+<!-- CORS appears to block the fonts from loading at this link -->
+<link
+  rel="stylesheet"
+  data-entry-name="style.css"
+  href="https://prod-va-gov-assets.s3-us-gov-west-1.amazonaws.com/generated/style.css"
+/>
```

Comment:
is this how we pick up styles in storybook?

Reply:
so there's a few different ways styles can be added to storybook.

- styles can be included in the component files themselves (either imported or css-in-js) [not really doing this in hte repo that i could see]. this is the most common approach when styles live in the repo as well.
- they can be included for all stories inside of `.storybook/preview.js` [this is where i import the local global styles we do have in the repo itself]
- this `preview-head.html` serves as an extension of sorts to the preview.js file mentioned above, it modifies the head inside of the <iframe> for all component stories. because we are using these external stylesheets, i added them here.